.travis.yml

```diff
@@ -14,17 +23,157 @@ before_script:
   - mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptest@localhost;"
   - mysql -uroot -e "CREATE USER otptest2@localhost IDENTIFIED BY 'otptest2';"
   - mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptest2@localhost;"
-  - mysql -uroot -e "CREATE USER otptestssl@localhost IDENTIFIED BY 'otptestssl';"
-  - mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptestssl@localhost REQUIRE SSL;"
+  - (mysql -uroot -e "CREATE USER otptestssl@localhost IDENTIFIED BY 'otptestssl' REQUIRE SSL;" &&
+    mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptestssl@localhost;") ||
+    (mysql -uroot -e "CREATE USER otptestssl@localhost IDENTIFIED BY 'otptestssl';" &&
+    mysql -uroot -e "GRANT ALL PRIVILEGES ON otptest.* TO otptestssl@localhost REQUIRE SSL;")
 script: 'make tests'
-otp_release:
-  - 22.0
-  - 21.1
-  - 21.0
-  - 20.0
-  - 19.0
-  - 18.3
-  - 17.3
-  - 17.0
-  - R16B03-1
-
+jobs:
+  include:
+# OTP/22.0 probably not supported on precise
+#    - name: OTP/22.0+MySQL5.5
+#      dist: precise
```

Comment:
It's totally fine to skip these combinations. I think picking one MySQL version for each OTP release would be sufficient, to keep the number of test runs to about the same as we have already.

Reply:
Oh, btw, I wanted to ask why those OTP releases in particular were picked for testing?

Reply:
No particular reason. I thought NN.0 would be good in general, but maybe the latest minor version per major version would make more sense.

Another option would be to pick the versions included of various (old) releases of Debian and other distros.

We can change them. Do you have a suggestion?

Reply:
Hm, I suggest we go for the latest minor versions (usually, the *.3s for past releases, and the respective latest minor for the latest major release), where most of the bugs of that major version have supposedly been fixed, so I'd expect developers to converge on them over time. If there are any particular releases containing a bug that affected us and that we had to work around, we may want to include those as well.

Reply:
OK! Only keep #85 in mind then.